Frameworks/GLKit/GLKViewController.mm

```diff
@@ -91,13 +101,12 @@ - (void)viewWillDisappear:(BOOL)animated {
 }
 
 - (BOOL)_renderFrame {
-    [_calendar setToNow];
-    WFDateTime* dt = [_calendar getDateTime];
-    int64_t frameTime = dt.universalTime;
+    _calendar.SetToNow();
+    WF::DateTime frameTime = _calendar.GetDateTime();
 
-    _timeSinceFirstResume = static_cast<NSTimeInterval>(frameTime - _firstStart) * 0.0000001;
-    _timeSinceLastResume = static_cast<NSTimeInterval>(frameTime - _lastStart) * 0.0000001;
-    _timeSinceLastUpdate = static_cast<NSTimeInterval>(frameTime - _lastFrame) * 0.0000001;
+    _timeSinceFirstResume = timeSpanToInterval(frameTime - _firstStart);
+    _timeSinceLastResume = timeSpanToInterval(frameTime - _lastStart);
+    _timeSinceLastUpdate = timeSpanToInterval(frameTime - _lastFrame);
```

Comment:
I agree that use of WinRT is not really required here.

I'm not sure I understand the concern about the system clock, though. Are you suggesting that the interval calculation would be wrong if the user moves, e.g., into a different time zone while the system is running?